kas/meta-coral.yml

```diff
@@ -41,3 +42,5 @@ local_conf_header:
     DISTRO_ARCH = "amd64"
   ccache: |
     USE_CCACHE = "1"
+  kernel: |
+    KERNEL_NAME = "${DISTRO_ARCH}"
```

Comment:
Only works for amd64 and arm64...

Reply:
Is that a problem here? This file will only be used locally for testing and in case we add support for further architectures, we still can overwrite the kernel entry.

Reply:
Just wanted to point this out, didn't look at the context in which this is used.